api/client.go

```diff
@@ -5,10 +5,11 @@ import (
 )
 
 type Client interface {
-	ListIPs(ctx context.Context) ([]IP, error)
+	ListIPs(context.Context) ([]IP, error)
```

Comment:
[nitpicking] Why ? You name the params on every other method.

Reply:
By default, we shouldn't include the variable name if it doesn't add any valuable information. For some methods, it adds value. For instance:

```go
AddIP(context.Context, string, AddIPParams) (IP, error)
```

This is not completely clear for me what the `string` parameter should be. Hence I keep the variable name which makes it way clearer:

```go
AddIP(ctx context.Context, ip string, params AddIPParams) (IP, error)
```

But for `ListIPs`, this is really clear what the parameter should contain.